packages/plugin-local-storage/README.md

```diff
@@ -0,0 +1,37 @@
+# ima-plugin-local-storage
+
+This is the local-storage plugin for the IMA.js application. 
+You can find the IMA.js skeleton application at
+<https://github.com/seznam/IMA.js-skeleton> or visit our site
+[imajs.io](https://imajs.io).
+
+## Installation
+
+```bash
+npm install ima-plugin-local-storage --save
+```
+
+```javascript
+// /app/build.js
+
+var vendors = {
+	common: [
+        'ima-plugin-local-storage'
+	]
+};
+```
+```javascript
```

Comment:
This part is not need.

Reply:
I've updated REDAME.md so it should be fixed now.